.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+python:
+- '3.6'
+install:
+- pip install .
+script:
+- python -m unittest -v test.py
+deploy:
+  provider: pypi
+  on:
+    tags: true
+  user: jessebrennan
```

Comment:
Please change to use user: `travis_integration_test_CGP`
The AWS keys for this user are now in Secrets Manager, with the secret name:
`admin/user/keys/aws/travis_integration_test_CGP`

Please let me know if this works for you!

Reply:
I'm using that bucket and credentials for AWS already. The user you see here is the PyPI account to which it is deployed.